tests/integration/pilot/analysis/analysis_test.go

```diff
@@ -87,12 +91,81 @@ spec:
 `)
 			// Status should no longer report error
 			retry.UntilSuccessOrFail(t, func() error {
-				return expectStatus(t, ctx, ns, false)
+				return expectVirtualServiceStatus(t, ctx, ns, false)
 			})
 		})
 }
 
-func expectStatus(t *testing.T, ctx resource.Context, ns namespace.Instance, hasError bool) error {
+func TestWorkloadEntryUpdatesStatus(t *testing.T) {
+	framework.NewTest(t).
+		Features(features.Usability_Observability_Status).
+		Run(func(ctx framework.TestContext) {
+			ns := namespace.NewOrFail(t, ctx, namespace.Config{
+				Prefix:   "default",
+				Inject:   true,
+				Revision: "",
+				Labels:   nil,
+			})
+			// Make a workloadentry
+			ctx.Config().ApplyYAMLOrFail(t, ns.Name(), `
+apiVersion: networking.istio.io/v1alpha3
+kind: WorkloadEntry
+metadata:
+  name: vm-1
+spec:
+  address: 2.2.2.2
+  labels:
+    app: details-legacy
+    instance-id: vm1
+`)
+			// Test adding one condition
+			stat := &v1alpha1.IstioStatus{
+				Conditions: []*v1alpha1.IstioCondition{
+					{
+						Type: "Health",
+						Reason: "DontTellAnyoneButImNotARealReason",
+						Status: "True",
+					},
+					{
+						Type: "SomeRandomType",
+						Reason: "ImNotHealthSoDontTouchMe",
+						Status: "True",
+					},
+				},
+			}
+			ctx.Clusters().Default().Istio().NetworkingV1alpha3().WorkloadEntries(ns.Name()).UpdateStatus(context.TODO(), &clientnetworkingv1alpha3.WorkloadEntry{
+				Status:  *stat,
+
+			}, metav1.UpdateOptions{})
+			retry.UntilSuccessOrFail(t, func() error {
+				// should update
+				return expectWorkloadEntryStatus(t, ctx, ns, stat.Conditions)
```

Comment:
The purpose of this test is to check compatibility with the CRD Status work from 1.6.  As such, the flow should look like: 

1. Create Workload Entry
2. Check for condition reconciled=true
3. Update Status to includ conditino Healthy=true
4. Check that condition reconciled=true and healthy=true

This will prove that this mechanism won't interfere with existing conditions.

Reply:
where does reconciled come from ehre?

Reply:
https://github.com/istio/istio/blob/b25c7eb039dc422cb47fc6015f38bf9236dd5208/pilot/pkg/status/state.go#L243

Reply:
It looks like reconciled is not created when I directly call Create(). Do I need to create it some other way for status to be Reconciled?